add workaround for charset error when migrate data (#10974)
pingcap/docs: dm/dm-faq.md

    ```

- Upgrade DM to v2.0.7 or later versions.

> ## The load unit reports the `Unknown character set` error

Author: ```suggestion
## Why does the load unit report the `Unknown character set` error?
```

> ## The load unit reports the `Unknown character set` error

Author: Fixed in [7a3ee27](https://github.com/pingcap/docs/pull/10974/commits/7a3ee271b216780b8e016083758bc7688c4118bb)

> TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance.

Reviewer: ```suggestion
TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table schema during a full import. To bypass this error, you can create the table schema in the downstream in advance using the [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data.
```

> TiDB does not support all MySQL characters sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see .

Reviewer 2: ```suggestion
TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see [character sets supported by TiDB](/character-set-and-collation.md).
```
